### ISSUE TITLE
How to correct reboot

### ISSUE THREAD

**Issue by GrapeBaBa** [lorin/devstack-vm#15]
Hi,
When i vagrant halt and vagrant up again, seems the openstack service is not running normally.

**Comment by lorin** [comment]
After you do a `vagrant up`, try doing a `vagrant provision` and see if that resolves it for you.

**Comment by GrapeBaBa** [comment]
No, though the service is running, but when i login the horizon, a lot of error display
